PrivateHeaders/APPX/ZIP.h

```diff
@@ -182,6 +184,44 @@ namespace appx {
         }
     };
 
+    inline
+    bool
+    _IsAPPXFile(const std::string &inputFileName)
```

Comment:
Precedent is to not treat private methods specially. There's no documentation or formal rules, hence this being a "nit".